src/Tribe/Subscribe/Purchase.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Tribe\HubSpot\Subscribe;
+
+use SevenShores\Hubspot\Factory;
+
+/**
+ * Class Connection
+ *
+ * @package Tribe\HubSpot\API
+ */
+class Purchase {
+
+	/**
+	 * Setup Hooks for OAuth
+	 *
+	 * @since 1.0
+	 *
+	 */
+	public function hook() {
+		add_action( 'event_ticket_woo_attendee_created', [ $this, 'connect' ], 10, 4 );
+	}
+
+	/**
+	 * Connect to Creation of an Attendee.
+	 *
+	 * @since TBD
+	 *
+	 * @param int    $attendee_id ID of attendee ticket.
+	 * @param int    $post_id     ID of event.
+	 * @param object $order       WooCommerce order.
+	 * @param int    $product_id  WooCommerce product ID.
+	 */
+	public function connect( $attendee_id, $post_id, $order, $product_id ) {
+
+		if ( ! $access_token = tribe( 'tickets.hubspot.api' )->is_ready() ) {
```

Comment:
I'm a fan of short and simple, but if we set tribe() to a var and phpdoc it, we future-proof our own development mistakes and make it much easier to work with an IDE.

Reply:
Plus we can reuse the API object below

Reply:
Made the change.